App/src/main/java/vut/fit/gja2023/app/validator/CsvValidator.java

```diff
@@ -0,0 +1,25 @@
+package vut.fit.gja2023.app.validator;
+
+import java.util.Arrays;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CsvValidator {
+    
+    public void validate(String[] row, int[] columnOrder) {
+        if (row == null || columnOrder == null) {
+            throw new NullPointerException();
+        }
+        
```

Comment:
hm pride mi divne vyhadzovat NullPointerException ked vlastne k null pointer exception nedoslo 😄 radsej znova IllegalArgumentException, v message napisat nieco ze row alebo column nemozu byt null, ak to chches vediet rozlisovat, tak mozno aj extend na IllegalArgumentException, a spravit si vlastnu NullArgumentException, ako chces